routes/celebrities.routes.js

```diff
@@ -0,0 +1,40 @@
+const { Router } = require("express");
+const router = require("express").Router();
+const Celebrity = require("../models/Celebrity.model");
+
+//const router = require("express").Router();
+
+// all your routes here
+
+router.get('/showcelebrities',(req,res)=>{
+    
+    Celebrity.find()
+    .then((allCelebrities)=>{
+        console.log(allCelebrities)
+        res.render('celebrities',{allCelebriteis})
```

Comment:
Aqui você não fez o push da versão nova

```suggestion
        res.render('celebrities',{allCelebrities})
```